apt-packages.rst

```diff
@@ -0,0 +1,80 @@
+.. post:: May 19, 2021
+   :tags: feature, builders
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      We have deployed a long awaited feature:
+      the ability to install custom operative system packages.
+
+Install custom operative system packages
+========================================
+
+We are thrilled to announce that now Read the Docs users
+can declare custom operative system packages in their project configuration
+that will get installed in our Ubuntu-based builders using ``apt``.
+This has been a long awaited feature,
+and we think it will simplify the configuration of many projects,
+especially scientific ones.
+
+Previous solutions
+------------------
+
+The Ubuntu images used by our builders
+contain `lots of preinstalled system
+packages <https://github.com/readthedocs/readthedocs-docker-images/blob/8e4f035c219307e30f5e3326c3c8271cee4f2631/Dockerfile#L15-L131>`_
+that we ship to all the projects
+to make the most common use cases possible.
+This includes compilers, development headers of common libraries, and others.
+
+However, on one hand this makes our images way bigger than necessary,
+and it is still not enough to solve 100 % of the use cases.
+Fortunately, Read the Docs has supported the `conda package
+manager <https://docs.readthedocs.io/en/stable/guides/conda.html>`_
+for a long time already,
+and this has allowed users with special needs to add
+any non-Python libraries they needed.
+Still, interoperability between conda and pip is not perfect,
+and for users that are not used to conda this could feel like a hack.
+
+New ``apt_packages`` configuration
+----------------------------------
+
+To overcome all these problems, we have added a new configuration value,
+``build.apt_packages``, that receive a list of APT packages
+that will be installed in our Ubuntu-based images.
+`Our configuration
+documentation <https://docs.readthedocs.io/en/stable/config-file/v2.html#build-apt-packages>`_
+contains a simple example:
+
+.. code-block:: yaml
+
+   build:
+     image: latest
+     apt_packages:
+       - libclang
+       - cmake
+
```

Comment:
We should mention being on Ubuntu 18.04 somewhere, with a note about upgrading soonish.

Reply:
Done!

Reply:
Ha, near future is maybe a bit strong, but we'll see :)